prow/plugins/trigger/pr.go

```diff
@@ -154,6 +157,8 @@ func welcomeMsg(ghc githubClient, trigger *plugins.Trigger, pr github.PullReques
 
 I'm waiting for a [%s](https://github.com/orgs/%s/people) %smember to verify that this patch is reasonable to test. If it is, they should reply with ` + "`/ok-to-test`" + ` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](%s) to skip this step.
 
+Once this PR has been blessed by a member with ` + "`/ok-to-test`" + `, this comment will stay and the new status will be reflected by a label.
+
```

Comment:
I am not a native speaker, so please review this comment.

Reply:
Maybe

```
Once the patch is verified, the new status will be reflected by the `ok-to-test` label
```